### ISSUE TITLE
Best way to open config files?

### ISSUE THREAD

**Issue by garethrees** [bbatsov/rails-style-guide#44]
In my `config/deploy.rb` I have some configuration variables stored in a YAML file.

What's the best way to load this in? Currently I have:

```ruby
CONFIG = YAML.load(File.open('config/deploy/deploy.yml'))

# >> File.open('config/deploy/deploy.yml')
# => #<File:config/deploy/deploy.yml>
```

This doesn't load a full path though, so I could do:

```ruby
CONFIG = YAML.load(File.open(Rails.root.join('config/deploy', 'deploy.yml')))

# >> File.open(Rails.root.join('config/deploy', 'deploy.yml'))
# => #<File:/Users/gareth/Code/repo/config/deploy/deploy.yml>
```

Is there a "standard" way of doing this? What do you guys do?

Thanks

**Comment by bbatsov** [comment]
@vestimir Guess we can mention this is the guide.

**Comment by vestimir** [comment]
I'll send a PR later today ;)

**Comment by garethrees** [comment]
Fixed by #136 :+1:
